docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -398,16 +431,22 @@
             <colspec />
 
             <tbody>
+              <row>
+                <entry>-aoption[=value]</entry>
+
+                <entry>Set an application option</entry>
+              </row>
+
               <row>
                 <entry>--allow=str</entry>
 
                 <entry>Allow use of named feature. (e.g., cpp, pipe,
-                all)<para></para><para><emphasis role="bold">cpp</emphasis>:
-                Allow embedded code within ECL (e.g., c++, JAVA, Javascript,
-                Python, R, etc.)</para><para><emphasis
-                role="bold">pipe</emphasis>: Allow the PIPE command to send
-                data to an external program.</para><para><emphasis
-                role="bold">all</emphasis>: Allow all features</para></entry>
+                all)<para><emphasis role="bold">cpp</emphasis>: Allow embedded
+                code within ECL (e.g., c++, JAVA, Javascript, Python, R,
+                etc.)</para><para><emphasis role="bold">pipe</emphasis>: Allow
+                the PIPE command to send data to an external
+                program.</para><para><emphasis role="bold">all</emphasis>:
+                Allow all features</para></entry>
```

Comment:
userECL: Allow code that is not found via the ecl include paths
datafile: Access to datafiles from ecl.
extern: access to an external service function

There is also a --allowsigned option which only allows access to the feature from signed code.